py3status/modules/wifi.py

```diff
@@ -73,6 +73,8 @@ class Py3status:
     signal_bad = 29
     signal_degraded = 49
     use_sudo = False
+    _ssid = None
+    _max_bitrate = 0
```

Comment:
I think your py3status install is screwed somehow, as post_config_hook() seems to not be being run. but also you are running a user module not the core one.